Python/compile.c

```diff
@@ -1170,6 +1178,9 @@ compiler_add_o(struct compiler *c, PyObject *dict, PyObject *o)
 {
     PyObject *v;
     Py_ssize_t arg;
+    if (!c->c_emit_bytecode) {
+        return 1;
```

Comment:
`1` is a valid value. It is not correct to return it unless `dict[o] == 1`.

Reply:
I will move the guards one level up then, unless there is a valid OK value here that does not collide with the semantics of other return codes.

Reply:
Check out 0662108. Not sure there is a better way (or a better return value). If you can think of a better way, please advice :)